test/X25519KeyAgreementKey2020.spec.js

```diff
@@ -117,12 +119,19 @@ describe('X25519KeyAgreementKey2020', () => {
     });
 
     it('should export both public and private key', async () => {
-      const key = await X25519KeyAgreementKey2020.generate();
+      const key = await X25519KeyAgreementKey2020.generate({
+        controller: 'did:example:1234'
+      });
+      const pastDate = new Date(2020, 11, 17).toISOString()
+        .replace(/\.[0-9]{3}/, '');
+      key.revoked = pastDate;
 
       const exported = key.export({publicKey: true, privateKey: true});
-      expect(exported).to.have.keys([
-        'id', 'controller', 'type', 'publicKeyMultibase', 'privateKeyMultibase'
-      ]);
+      expect(exported).to.have.keys(['id', 'type', 'controller',
+        'publicKeyMultibase', 'privateKeyMultibase', 'revoked']);
```

Comment:
sorry, missed this before: style, trying to eyeball 80 char limit.
```suggestion
      expect(exported).to.have.keys([
        'id', 'type', 'controller', 'publicKeyMultibase',
        'privateKeyMultibase', 'revoked'
      ]);
```